ocs_ci/deployment/hub_spoke.py

```
@@ -5192,59 +5263,97 @@ def _cleanup_stale_oidc_providers(self):
 
-        success = True
-        prefix = f"{self.infra_id}/"
-
         try:
-            paginator = self.s3_client.get_paginator("list_objects_v2")
-            objects_to_delete = []
+            self.s3_client.head_bucket(Bucket=self.oidc_bucket_name)
+        except ClientError:
+            logger.info(
+                f"OIDC bucket '{self.oidc_bucket_name}' does not exist, "
+                "nothing to clean up"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)